osc-dm-product-srv/app/startd.sh

```diff
@@ -72,6 +71,7 @@ echo "CONFIG_NAME:  $CONFIG_NAME   <--- This is data product configuration in th
 echo "DATA_DIR:     $DATA_DIR      <--- This is data product data directory (/app/dataproducts) directory"
 echo " "
 
+echo "${DATA_DIR}/dataproducts"
```

Comment:
should probably add some context to what is being echoed here

```
echo "using dataproducts directory: ${DATA_DIR}/dataproducts"
```

Reply:
This was left in from testing, removed